qdrant_client/uploader/grpc_uploader.py

```diff
@@ -49,7 +50,11 @@ def upload_batch_grpc(
             )
             break
         except Exception as e:
-            logging.warning(f"Batch upload failed {attempt + 1} times. Retrying...")
+            show_warning(
+                message=f"Batch upload failed {attempt + 1} times. Retrying...",
+                category=UserWarning,
+                stacklevel=1,
```

Comment:
seems that it should be 8

Reply:
This one does not reproduce show_warning message, but raises RuntimeError("Thread unexpectedly terminated") if the qdrant is stopped